src/constants/acbs-id.constant.ts

```diff
@@ -8,4 +8,7 @@ export const ACBSID = {
     PREFIX: '0000',
     REGEX: /^0{4}\d{6}$/,
   },
+  LOAN_ID: {
+    REGEX: /^\d{10}$/, // TODO APIM-128: Should this have a prefix?
```

Comment:
@avaitonis do you know if loan ids should have a prefix?

Reply:
Hi, after testing looks like 000 is safe prefix for DEV and PROD.

Reply:
Okay, thanks!